adhocracy4/polls/assets/PollQuestion.jsx

```diff
@@ -33,9 +30,6 @@ class Question extends React.Component {
   handleSubmit (event) {
     event.preventDefault()
 
-    const voteCountText = django.gettext('Vote counted')
-    const voteNoCountText = django.gettext('Vote has not been counted due to a server error.')
-
     if (this.state.question.isReadOnly) {
```

Comment:
@khamui we have to have translations in react in variables otherwise they are not always picked up by `make messages` can be changed later